---
layout: page
title: "How do I get a sub-section of a vcf file? | 1000 Genomes"
permalink: /faq/how-do-i-get-sub-section-vcf-file/
---
    <div id="main" class="column grid-13 ">
  	  <div id="content">
  	    <div id="content-inner">

          <div class="breadcrumb"><a href="/">Home</a> › <a href="/faq">FAQ</a> › </div>
          
          
                    
          
          <div id="main-content" class="region clear-block">
            
<div id="node-758" class="node node-type-faq-question"><div class="node-inner onethousand_clean">

  
      <h2 class="title">
      <a href="/faq/how-do-i-get-sub-section-vcf-file" title="How do I get a sub-section of a vcf file?">How do I get a sub-section of a vcf file?</a>
    </h2>
  
  
      <div class="meta">
      
              <div class="terms terms-inline"><ul class="links inline"><li class="taxonomy_term_2 first"><a href="/category/data-access" rel="tag" title="">Data Access</a></li>
<li class="taxonomy_term_18"><a href="/category/tabix" rel="tag" title="">tabix</a></li>
<li class="taxonomy_term_17"><a href="/category/tools" rel="tag" title="">tools</a></li>
<li class="taxonomy_term_6"><a href="/category/variants" rel="tag" title="">variants</a></li>
<li class="taxonomy_term_10 last"><a href="/category/vcf" rel="tag" title="">vcf</a></li>
</ul></div>
          </div>
  
  <div class="content">
    <div class="field field-type-text field-field-faq-answer">
      <div class="field-label">Answer:&nbsp;</div>
    <div class="field-items">
            <div class="field-item odd">
                    
<p>There are two ways to get subsections of our VCF files.</p>
<p>The first is to use the&nbsp;<a href="http://browser.1000genomes.org/tools.html">Data Slicer tool</a>&nbsp;from our&nbsp;<a href="http://browser.1000genomes.org/">browser</a>&nbsp;which is&nbsp;<a href="/data-slicer">documented here</a>. This tool gives you a web interface requesting the url of any vcf file and the genomic location you wish to get a sub slice for. This tool also works for BAM files. This tool also allows you to filter the file for particular individuals or populations if you also provide a panel file.</p>
<p>The second method is using <a href="http://sourceforge.net/projects/samtools/files/tabix/">tabix</a> on the command line. e.g&nbsp;</p>
<pre>tabix -h <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20100804/ALL.2of4intersection.20100804.genotypes.vcf.gz" title="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20100804/ALL.2of4intersection.20100804.genotypes.vcf.gz">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20100804/ALL.2of4intersection.20100804.genotypes.vcf.gz</a> 2:39967768-39967768</pre>
<p>The VCF format and a c++ and perl tool set for vcf files can be found at <a href="http://vcftools.sourceforge.net/">vcftools on sourceforge</a></p>
<p>Please note all our vcf files using straight intergers and X/Y for their chromosome names in the Ensembl style rather than using chr1 in the UCSC style. If you request a subsection of a vcf file using a chromosome name in the style chrN as shown below it will not work.</p>
<pre>  tabix -h <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20100804/ALL.2of4intersection.20100804.genotypes.vcf.gz" title="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20100804/ALL.2of4intersection.20100804.genotypes.vcf.gz">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20100804/ALL.2of4intersection.20100804.genotypes.vcf.gz</a> chr2:39967768-39967768</pre>
        </div>
        </div>
</div>
<div class="field field-type-nodereference field-field-faq-related">
      <div class="field-label">Related questions:&nbsp;</div>
    <div class="field-items">
            <div class="field-item odd">
                    <a href="/faq/can-i-convert-vcf-files-plinkped-format">Can I convert vcf files to plink/ped format?</a>        </div>
              <div class="field-item even">
                    <a href="/faq/what-depth-coverage-your-phase1-variants">What is the depth of coverage of your Phase1 variants</a>        </div>
              <div class="field-item odd">
                    <a href="/faq/can-i-find-genomic-position-list-dbsnp-rs-numbers-0">Can I find the genomic position for a list of dbSNP rs numbers</a>        </div>
              <div class="field-item even">
                    <a href="/faq/can-i-get-genotypes-specific-individualpopulation-your-vcf-files">Can I get genotypes for a specific individual/population from your vcf files</a>        </div>
              <div class="field-item odd">
                    <a href="/faq/can-i-get-individual-genotype-information-browser1000genomesorg">Can I get individual genotype information from browser.1000genomes.org</a>        </div>
        </div>
</div>
  <div id="book-navigation-503" class="book-navigation">
    
        <div class="page-links clear-block">
              <a href="/faq/how-do-i-get-sub-section-bam-file" class="page-previous" title="Go to previous page">‹ How do I get a sub-section of a bam file</a>
                    <a href="/faq" class="page-up" title="Go to parent page">up</a>
                    <a href="/faq/1000-genomes-data-available-genome-browsers" class="page-next" title="Go to next page">Is the 1000 genomes data available in genome browsers? ›</a>
          </div>
    
  </div>
  </div>

  
</div></div> <!-- /node-inner, /node -->          </div> <!-- /#main-content -->

          
		
		</div> <!-- // #content-inner -->
	  </div> <!-- //#content -->
	</div> <!-- //end #main -->

    
        <div id="sidebar-right" class="column sidebar region grid-3">
	    <div id="sidebar-right-inner">
          <div id="block-user-1" class="block block-user region-odd odd region-count-1 count-1"><div class="block-inner">

      <h2 class="title">Navigation</h2>
  
  <div class="content">
    <ul class="menu"><li class="leaf first last"><a href="/faq">Frequently Asked Questions</a></li>
</ul>  </div>

  
</div></div> <!-- /block-inner, /block -->
      </div>
    </div> <!--//end #sidebar-right-inner -->
    

