---
layout: page
title: "How can I get the allele frequency of my variant? | 1000 Genomes"
permalink: /faq/how-can-i-get-allele-frequency-my-variant/
---
    <div id="main" class="column grid-13 ">
  	  <div id="content">
  	    <div id="content-inner">

          <div class="breadcrumb"><a href="/">Home</a> › <a href="/faq">FAQ</a> › </div>
          
          
                    
          
          <div id="main-content" class="region clear-block">
            
<div id="node-868" class="node node-type-faq-question"><div class="node-inner onethousand_clean">

  
      <h2 class="title">
      <a href="/faq/how-can-i-get-allele-frequency-my-variant" title="How can I get the allele frequency of my variant?">How can I get the allele frequency of my variant?</a>
    </h2>
  
  
      <div class="meta">
      
              <div class="terms terms-inline"><ul class="links inline"><li class="taxonomy_term_2 first"><a href="/category/data-access" rel="tag" title="">Data Access</a></li>
<li class="taxonomy_term_17"><a href="/category/tools" rel="tag" title="">tools</a></li>
<li class="taxonomy_term_6"><a href="/category/variants" rel="tag" title="">variants</a></li>
<li class="taxonomy_term_10 last"><a href="/category/vcf" rel="tag" title="">vcf</a></li>
</ul></div>
          </div>
  
  <div class="content">
    <div class="field field-type-text field-field-faq-answer">
      <div class="field-label">Answer:&nbsp;</div>
    <div class="field-items">
            <div class="field-item odd">
                    
<p>Our vcf files contain global and super population alternative allele frequencies. You can see this is our<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/">&nbsp;most recent release</a>. For multi allelic variants, each alternative allele frequency is presented in a comma separated list.</p>
<p>An example info column which contains this information looks like</p>
<pre>   1 15211 rs78601809 T G 100 PASS AC=3050;AF=0.609026;AN=5008;NS=2504;DP=32245;EAS_AF=0.504;AMR_AF=0.6772;AFR_AF=0.5371;EUR_AF=0.7316;SAS_AF=0.6401;AA=t|||;VT=SNP
</pre>
<p>If you want population specific allele frequencies you have three options.</p>
<p>For a single snp you can look at the&nbsp;<a href="http://browser.1000genomes.org/Homo_sapiens/Variation/Population?r=1:14711-15711;source=dbSNP;v=rs78601809;vdb=variation;vf=22041749">population genotype page for a snp in our browser</a>. This gives you piecharts and a table for a single site.</p>
<p>For a genomics region you can use our&nbsp;<a href="/allele-frequency-calculator-documentation">allele frequency calculator</a>&nbsp;tool which gives a set of allele frequencies for selected populations&nbsp;</p>
<p>If you would like sub population allele frequences for a whole file, you are best to use the vcftools command line tool.</p>
<p>This is done using a combination of two&nbsp;<a href="http://vcftools.sourceforge.net/">vcftools&nbsp;</a>&nbsp;commands called&nbsp;<a href="http://vcftools.sourceforge.net/perl_module.html#vcf-subset">vcf-subset</a>&nbsp;and&nbsp;<a href="http://vcftools.sourceforge.net/perl_module.html#fill-an-ac">fill-an-ac</a></p>
<p>An example command set using files from our&nbsp;<a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/integrated_call_sets/">phase1 release</a>&nbsp;would look like&nbsp;</p>
<pre>    grep CEU integrated_call_samples.20101123.ALL.panel | cut -f1 &gt; CEU.samples.list
    <br />
    vcf-subset -c CEU.samples.list ALL.chr13.integrated_phase1_v3.20101123.snps_indels_svs.genotypes.vcf.gz | fill-an-ac&nbsp;|
    bgzip -c &gt; CEU.chr13.phase1.vcf.gz
</pre>
<p>Once you have this file you can calculate your frequency by dividing AN (allele number) by AC (allele count)</p>
<p>Please note that some early vcf files from the main project used LD information and other variables to help estimate the allele frequency. This means in these files the AF does not always equal AC/AN. The phase1 and phase 3 releases, AC/AN should always match the allele frequency quoted.</p>
        </div>
        </div>
</div>
<div class="field field-type-nodereference field-field-faq-related">
      <div class="field-label">Related questions:&nbsp;</div>
    <div class="field-items">
            <div class="field-item odd">
                    <a href="/faq/how-do-i-get-sub-section-vcf-file">How do I get a sub-section of a vcf file?</a>        </div>
              <div class="field-item even">
                    <a href="/faq/can-i-get-genotypes-specific-individualpopulation-your-vcf-files">Can I get genotypes for a specific individual/population from your vcf files</a>        </div>
              <div class="field-item odd">
                    <a href="/faq/can-i-convert-vcf-files-plinkped-format">Can I convert vcf files to plink/ped format?</a>        </div>
        </div>
</div>
  <div id="book-navigation-503" class="book-navigation">
    
        <div class="page-links clear-block">
              <a href="/faq/how-are-your-alignments-generated" class="page-previous" title="Go to previous page">‹ How are your alignments generated?</a>
                    <a href="/faq" class="page-up" title="Go to parent page">up</a>
                    <a href="/faq/how-many-individuals-will-be-sequenced" class="page-next" title="Go to next page">How many individuals will be sequenced? ›</a>
          </div>
    
  </div>
  </div>

  
</div></div> <!-- /node-inner, /node -->          </div> <!-- /#main-content -->

          
		
		</div> <!-- // #content-inner -->
	  </div> <!-- //#content -->
	</div> <!-- //end #main -->

    
        <div id="sidebar-right" class="column sidebar region grid-3">
	    <div id="sidebar-right-inner">
          <div id="block-user-1" class="block block-user region-odd odd region-count-1 count-1"><div class="block-inner">

      <h2 class="title">Navigation</h2>
  
  <div class="content">
    <ul class="menu"><li class="leaf first last"><a href="/faq">Frequently Asked Questions</a></li>
</ul>  </div>

  
</div></div> <!-- /block-inner, /block -->
      </div>
    </div> <!--//end #sidebar-right-inner -->
    

