---
layout: page
title: "What is the depth of coverage of your Phase1 variants | 1000 Genomes"
permalink: /faq/what-depth-coverage-your-phase1-variants/
---
    <div id="main" class="column grid-13 ">
  	  <div id="content">
  	    <div id="content-inner">

          <div class="breadcrumb"><a href="/">Home</a> › <a href="/faq">FAQ</a> › </div>
          
          
                    
          
          <div id="main-content" class="region clear-block">
            
<div id="node-828" class="node node-type-faq-question"><div class="node-inner onethousand_clean">

  
      <h2 class="title">
      <a href="/faq/what-depth-coverage-your-phase1-variants" title="What is the depth of coverage of your Phase1 variants">What is the depth of coverage of your Phase1 variants</a>
    </h2>
  
  
      <div class="meta">
      
              <div class="terms terms-inline"><ul class="links inline"><li class="taxonomy_term_2 first"><a href="/category/data-access" rel="tag" title="">Data Access</a></li>
<li class="taxonomy_term_7"><a href="/category/data-analysis" rel="tag" title="">Data Analysis</a></li>
<li class="taxonomy_term_117"><a href="/category/frequently-asked-questions/data-slicer" rel="tag" title="">data slicer</a></li>
<li class="taxonomy_term_142"><a href="/category/frequently-asked-questions/depth" rel="tag" title="">depth</a></li>
<li class="taxonomy_term_10 last"><a href="/category/vcf" rel="tag" title="">vcf</a></li>
</ul></div>
          </div>
  
  <div class="content">
    <div class="field field-type-text field-field-faq-answer">
      <div class="field-label">Answer:&nbsp;</div>
    <div class="field-items">
            <div class="field-item odd">
                    
<p>The&nbsp;<a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/integrated_call_sets/">Phase1 integrated variant</a>&nbsp;set does not report the depth of coverage for each individual at each site. We instead report genotype likelihoods and dosage. If you would like to see depth of coverage numbers you will need to calculate them directly.</p>
<p>The&nbsp;<a href="http://code.google.com/p/bedtools/">bedtools</a>&nbsp;suite provides a method to do this.&nbsp;</p>
<p><strong>genomeCoverageBed</strong>&nbsp;is a tool which can provide a bed file which specifies coverage for every base in the genome and&nbsp;<strong>i</strong><strong>ntersectBed</strong>&nbsp;which will provide an intersection between two vcf/bed/bam files</p>
<p>These commands also require&nbsp;<a href="http://samtools.sourceforge.net/">samtools</a>,&nbsp;<a href="http://sourceforge.net/projects/samtools/files/tabix/">tabix</a>&nbsp;&nbsp;and&nbsp;<a href="http://vcftools.sourceforge.net/">vcftools</a>&nbsp;to be installed</p>
<p>an example set of commands would be</p>
<p>&nbsp;</p>
<pre>samtools view -b  <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data/HG01375/alignment/HG01375.mapped.ILLUMINA.bwa.CLM.low_coverage.20120522.bam" title="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data/HG01375/alignment/HG01375.mapped.ILLUMINA.bwa.CLM.low_coverage.20120522.bam">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data/HG01375/alignment/HG01375.mapped.ILLUMINA.bwa.CLM.low_coverage.20120522.bam</a> 2:1,000,000-2,000,000 | genomeCoverageBed -ibam stdin -bg &gt; coverage.bg </pre>
<p>&nbsp;</p>
<p>This command gives you a bedgraph file of the coverage of the HG01375 bam between 2:1,000,000-2,000,000</p>
<p>&nbsp;</p>
<pre>tabix -h <a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/integrated_call_sets/ALL.chr2.integrated_phase1_v3.20101123.snps_indels_svs.genotypes.vcf.gz" title="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/integrated_call_sets/ALL.chr2.integrated_phase1_v3.20101123.snps_indels_svs.genotypes.vcf.gz">http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/integrated_call_sets/ALL.chr2.integrated_phase1_v3.20101123.snps_indels_svs.genotypes.vcf.gz</a> 2:1,000,000-2,000,000 | vcf-subset -c HG01375 | bgzip -c &gt; HG01375.vcf.gz</pre>
<p>&nbsp;</p>
<p>This command gives you the vcf file for 2:1,000,000-2,000,000 with just the genotypes for HG01375</p>
<p>To get the coverage for all those sites you would use</p>
<p>&nbsp;</p>
<pre>intersectBed -a HG01375.vcf.gz -b coverage.bg -wb &gt; depth_numbers.vcf</pre>
<p>You can find more information about bed file formats please see the&nbsp;<a href="http://genome.ucsc.edu/FAQ/FAQformat.html">UCSC File Formats Help</a></p>
<p>For more information you may wish to look at our documentation about&nbsp;<a href="/faq/how-do-i-get-slice-your-vcf-files">data slicing</a></p>
        </div>
        </div>
</div>
<div class="field field-type-nodereference field-field-faq-related">
      <div class="field-label">Related questions:&nbsp;</div>
    <div class="field-items">
            <div class="field-item odd">
                    <a href="/faq/how-do-i-get-sub-section-vcf-file">How do I get a sub-section of a vcf file?</a>        </div>
              <div class="field-item even">
                    <a href="/faq/can-i-convert-vcf-files-plinkped-format">Can I convert vcf files to plink/ped format?</a>        </div>
              <div class="field-item odd">
                    <a href="/faq/can-i-get-genotypes-specific-individualpopulation-your-vcf-files">Can I get genotypes for a specific individual/population from your vcf files</a>        </div>
        </div>
</div>
  <div id="book-navigation-503" class="book-navigation">
    
        <div class="page-links clear-block">
              <a href="/faq/what-completion-criteria-samples-project" class="page-previous" title="Go to previous page">‹ What is the completion criteria for samples in the project?</a>
                    <a href="/faq" class="page-up" title="Go to parent page">up</a>
                    <a href="/faq/what-difference-between-sequenceindex-and-analysissequenceindex" class="page-next" title="Go to next page">What is the difference between the sequence.index and the analysis.sequence.index ›</a>
          </div>
    
  </div>
  </div>

  
</div></div> <!-- /node-inner, /node -->          </div> <!-- /#main-content -->

          
		
		</div> <!-- // #content-inner -->
	  </div> <!-- //#content -->
	</div> <!-- //end #main -->

    
        <div id="sidebar-right" class="column sidebar region grid-3">
	    <div id="sidebar-right-inner">
          <div id="block-user-1" class="block block-user region-odd odd region-count-1 count-1"><div class="block-inner">

      <h2 class="title">Navigation</h2>
  
  <div class="content">
    <ul class="menu"><li class="leaf first last"><a href="/faq">Frequently Asked Questions</a></li>
</ul>  </div>

  
</div></div> <!-- /block-inner, /block -->
      </div>
    </div> <!--//end #sidebar-right-inner -->
    

