---
layout: single_section
title: "About"
permalink: /about2/
tags: About
redirect_from:
    - /node/3/
nav_buttons:
    - announcements
    - media
    - pilot_paper
    - contacts
---

# About IGSR and the 1000 Genomes Project

The 1000 Genomes Project, set up in 2008, was the first large-scale project to use genome wide sequencing to catalogue human variation and create open and freely available data sets, which have become widely used references. To preserve and extend the usability of the 1000 Genomes data following completion of the project in 2015, the International Genome Sample Resource (IGSR) was established with the following aims:

1. Ensure the future usability of the 1000 Genomes reference data
2. Incorporate additional published genomic data on the 1000 Genomes samples
3. Expand the data collection to include new populations not represented in the 1000 Genomes Project

## The 1000 Genomes Project

###Overview of the 1000 Genomes Project

The 1000 Genomes Project was established in 2008 to take advantage of developments in sequencing technology and undertake large scale population level sequencing to discover variation. This was made possible by the sharp reduction in the cost of sequencing which began in the mid 2000s and continues to today. It was the first project to sequence the genomes of a large number of people, to provide a comprehensive resource on human genetic variation. Data from the 1000 Genomes Project was quickly made available to the worldwide scientific community through the sequence archives hosted at [EMBL-EBI](http://www.ebi.ac.uk) and NCBI and through the project's own FTP site.

The goal of the 1000 Genomes Project was to find most genetic variants with frequencies of at least 1% in the populations studied.

Sequencing remained too expensive to deeply sequence the many samples being studied in the project. However, any particular region of the genome generally contains a limited number of haplotypes. Data was combined across samples to allow efficient detection of most of the variants in a region. The project planned to sequence each sample to about 4x coverage; at this depth sequencing can not provide the complete genotype of each sample, but can allow the detection of most variants with frequencies as low as 1%. The data from 2,504 samples was combined to allow highly accurate estimation (imputation) of the variants and genotypes for each sample, even where not seen directly by low coverage sequencing.

The contribution of the 1000 Genomes Project was summarised in [_Nature_](http://www.nature.com/nature/journal/v526/n7571/full/526052a.html) in the issue containing the final publications from the main project.

###Design of the 1000 Genomes Project

The Project was conducted in four stages: a pilot phase and three phases of the main project. In the main project, phases one and three produced data, with phase two concentrating on technical development.

####Pilot project

Three pilot studies provided data to inform the design of the full-scale project:

| Pilot | Purpose | Coverage | Strategy | Status
|:-:|:-:|:-:|:-:|:-:|
| 1 - low coverage | Assess strategy of sharing data across samples | 2-4X | Whole-genome sequencing of 180 samples | Sequencing completed October 2008 |
| 2 - trios | Assess coverage and platforms and centers | 20-60X | Whole-genome sequencing of 2 mother-father-adult child trios | Sequencing completed October 2008 |
| 3 - gene regions | Assess methods for gene-region-capture | 50X | 1000 gene regions in 900 samples | Sequencing completed June 2009 |

Data from the pilot projects was analyzed to determine whether the strategy of 4x coverage was adequate to meet the goals of the project. A paper describing the analyses of the pilot data and design of the full project was published in October 2010 in the journal _Nature_ and is available [here](/sites/1000genomes.org/files/docs/nature09534.pdf).

####Main project

Sequencing was carried out in phases one and three of the main project, with data releases and analysis corresponding to each. The final data freeze, associated with the third and final phase, took place on the 2nd May 2013. This data set represented the finalised data set upon which the phase three analysis was based and superseded previous data releases. During the course of the project, analysis methods were further developed and the phase three analysis replaces earlier versions.

The final data set contains data for 2,504 individuals from 26 populations. Low coverage sequence data is present for all individuals, with additional high coverage and exome sequence data being produced for some samples.

Analyses were conducted looking at both the comparatively simple short variations (up to 500 base pairs in length) and also structural variations. These analyses were published at the conclusion of the project in 2015 by [The 1000 Genomes Project Consortium](http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html) and [Sudmant _et al._](http://www.nature.com/nature/journal/v526/n7571/full/nature15394.html) respectively.

###1000 Genomes Project samples and data

The 1000 Genomes Project developed guidelines on ethical considerations for investigators doing sampling, outlined in the  [Informed Consent Background Document](http://www.1000genomes.org/sites/1000genomes.org/files/docs/Informed%20Consent%20Background%20Document.pdf) and the [Informed Consent Form Template](http://www.1000genomes.org/sites/1000genomes.org/files/docs/Informed%20Consent%20Form%20Template.pdf). All collections included in the Project followed these ethical guidelines and model informed consent language. The 1000 Genomes Project Steering Committee, with input from the Project's Samples and ELSI Group, made final decisions about which populations and sample sets to include in the Project.

Data from the 1000 Genomes Project is available without embargo, following the final publications from the project. Use of the data should be cited in the usual way, with current details available [here](http://www.1000genomes.org/faq/how-do-i-cite-1000-genomes-project). Additional details on data use can be found [here](https://github.com/igsr/1000Genomes_data_indexes/blob/master/IGSR_data_disclaimer.md).

The available data from the 1000 Genomes Project can be explored [here](http://www.1000genomes.org/data), alongside other data in IGSR. Cell lines and DNA are avaiable for all 1000 Genomes samples and can be obtained from the [Coriell Institute](https://catalog.coriell.org).

The samples for the 1000 Genomes Project mostly are anonymous and have no associated medical or phenotype data; for some of the populations the collectors have phenotype data but these data are not at Coriell and are not distributed.

###Key publications and documents from the 1000 Genomes Project

####October 2015

[1000 genomes main project paper](http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html)

[1000 geneomes structural variation paper](http://www.nature.com/nature/journal/v526/n7571/full/nature15394.html)

####October 2010

[1000 genomes pilot paper](http://www.nature.com/nature/journal/v467/n7319/full/nature09534.html)

####September 2007

[Report of the 1000 Genomes planning meeting](/sites/1000genomes.org/files/docs/1000Genomes-MeetingReport.pdf)

####Consent documents

[Informed Consent Background Document](http://www.1000genomes.org/sites/1000genomes.org/files/docs/Informed%20Consent%20Background%20Document.pdf)

[Informed Consent Form Template](http://www.1000genomes.org/sites/1000genomes.org/files/docs/Informed%20Consent%20Form%20Template.pdf)

##IGSR

As stated, IGSR was set up to ensure the future usability and accessibility of data from the 1000 Genomes Project and to extend the data set to include new data on the 1000 Genomes samples and new populations where sampling has been carried out in line with IGSR sampling principles.

###1. Ensuring the future usability of the 1000 Genomes reference data

In 2014, the [Genome Reference Consortium](http://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/) released an update of the human assembly, GRCh38. This update to the human reference assembly shows a significant improvement in the quantity of alternative loci represented. It now contains 178 genomic regions with associated alternative loci (2% of chromosomal sequence (61.9 Mb)). This is made up from 261 alternative loci (containing 3.6 Mb novel sequence relative to chromosomes). The GRC were also able to resolve more than 1000 issues from the previous version of the assembly.

Taking advantage of the alternative loci when identifying variation and calling genotypes is an important step in improving our ability to discover human variation. Currently very few tools can use the alternative loci data. IGSR has remapped the [phase 3 1000 Genomes data](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/) to GRCh38 in an alternative loci aware manner using [BWA mem](http://bio-bwa.sourceforge.net/). This provides the method development community with a source of alignments that can drive new methods forward, as well as providing the wider community with up to date alignments, ensuring everyone can benefit from the data in the context of the new assembly.

In addition, further sets of genomic sequence data are being aligned to GRCh38, with the [Platinum Genomes data from Illumina](http://www.illumina.com/platinumgenomes/) being the first new collection of data to be aligned.

###2. Incorporate published genomic data on the 1000 Genomes samples

The 1000 Genomes samples have proved a popular resource for molecular phenotyping experiments and investigating the associations between genetic variation and expression or measurements of epigenetic state. Large datasets have been generated on these samples by projects such as [GEUVADIS](http://www.geuvadis.org/web/geuvadis), who generated [RNA-Seq data](http://www.geuvadis.org/web/geuvadis/rnaseq-project) on the 1000 Genomes European samples and the YRI population, and [ENCODE](https://www.encodeproject.org/), who have carried out extensive assays on the [NA12878](https://www.encodeproject.org/search/?searchTerm=GM12878) cell line. Many other groups have also conducted assays on the 1000 Genomes samples. The IGSR would like to present all this information in a unified manner so the community can benefit from all the data which exists on these samples.

###3. Expand the data collection to include new populations

The IGSR recognises that the current 1000 Genomes Project samples do not reflect all populations. An important aim for IGSR is to expand the populations represented in the collection and ensure the available public data represents the maximum possible population diversity. This will ensure the 1000 Genomes dataset remains a valuable open resource for the community over the next five years. The IGSR will work with the groups who were unable to contribute samples to the 1000 Genomes Project before it finished sample collection and investigate collaborations with other groups to ensure the population diversity gaps are filled. You can find more details about this on our [sample collection principles page]().

Questions about any of the above can be sent to info@1000genomes.org.

