---
title: "Is the 1000 genomes data available in genome browsers?"
faq_tags:
  - data-access
  - ensembl
  - ucsc
  - variants
faq_related:
  - can-i-access-databases-associated-browser
  - are-there-any-scripts-or-apis-use-1000-genomes-data-sets
  - where-most-recent-release
---
                    
1000 genomes data sets are available at both [Ensembl](http://www.ensembl.org) and the [UCSC Genome Browser](http://genome.ucsc.edu/).

More information on accessing 1000 Genomes Project data in genome browsers can be found on the [Browser page](/1000-genomes-browsers).

Ensembl provides consequence information for the variants. The variants that are loaded into the Ensembl database and have consequence types assigned are displayed on the [Variation view](http://browser.1000genomes.org/Homo_sapiens/Variation/Mappings?db=core;r=6:74125388-74126388;v=rs311685;vdb=variation;vf=14071116).  Ensembl can also offer consequence predictions using their [Variant Effect Predictor (VEP)](http://browser.1000genomes.org/Homo_sapiens/UserData/UploadVariations?db=core).

You can see individual genotype information in the Ensembl browser by looking at the [Individual Genotypes](http://browser.1000genomes.org/Homo_sapiens/Variation/Individual?r=6:74125388-74126388;v=rs311685;vdb=variation;vf=14071116) section of the page from the menu on the left hand side.
