---
title: "Is the 1000 genomes data available in genome browsers?"
faq_tags:
  - data-access
  - ensembl
  - ucsc
  - variants
faq_related:
  - can-i-access-databases-associated-browser
  - are-there-any-scripts-or-apis-use-1000-genomes-data-sets
  - where-most-recent-release
---
                    
1000 genomes data sets are available at both [Ensembl](http://www.ensembl.org){:target="_blank"} and [UCSC Genome Browser](http://genome.ucsc.edu/){:target="_blank"}

These browsers get the 1000 genomes data from dbSNP so they may not always contain the most recent 1000 genomes variant release but the 1000 genomes project has its own version of [Ensembl](http://browser.1000genomes.org){:target="_blank"} which contains more recent variant sets. We also have a frozen version of our browser for the pilot data set at [http://pilotbrowser.1000genomes.org](http://pilotbrowser.1000genomes.org){:target="_blank"}

Ensembl can also provides consequence information for the variants. The variants that are loaded into the Ensembl database and have consequence types assigned and displayed on the [Variation view](http://browser.1000genomes.org/Homo_sapiens/Variation/Mappings?db=core;r=6:74125388-74126388;v=rs311685;vdb=variation;vf=14071116){:target="_blank"}.  Ensembl can also offer consequence predictions using their [Variant effect predictor](http://browser.1000genomes.org/Homo_sapiens/UserData/UploadVariations?db=core){:target="_blank"}.

You can also see individual genotype information in the Ensembl browser by looking at the [Individual Genotypes](http://browser.1000genomes.org/Homo_sapiens/Variation/Individual?r=6:74125388-74126388;v=rs311685;vdb=variation;vf=14071116){:target="_blank"} section of the page from the menu on the left hand side.
