---
layout: faq
title: "Are my snps found by the 1000 genomes project?"
faq_tags:
  - data-access
  - tools
faq_related:
  - can-i-get-genotypes-specific-individualpopulation-your-vcf-files
  - how-do-i-get-sub-section-vcf-file
  - what-do-names-your-variant-files-mean-and-what-format-are-files
---
                    
<p>There are two ways to discover if your variants are found in our variant list.</p>
<p>The first is to use our <a href="http://browser.1000genomes.org/index.html">Ensembl style browser</a>&nbsp;which allows you to search both by rs number and by genomic location. Our&nbsp;<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/browser/1000genomes_browser_main_project_20110521/The_1000_Genomes_Browser_Tutorial.ensembl_65.doc">tutorial</a>&nbsp;gives more detailed instructions.&nbsp;</p>
<p>The second is to use our <a href="http://www.1000genomes.org/data-slicer">Data Slicer</a> which can also be accessed from the browser <a href="http://browser.1000genomes.org/tools.html">tools menu</a>. This allows you to get specific regions of our VCF files. This can also be used in a <a href="http://www.1000genomes.org/faq/how-do-i-get-sub-section-vcf-file">command line manner</a>.</p>
