---
title: "Are the IGSR variants available in genome browsers?"
faq_tags:
  - data-access
  - ensembl
  - ucsc
  - variants
faq_related:
  - can-i-access-databases-associated-browser
  - can-i-query-igsr-programmatically
  - how-do-i-find-the-most-up-to-date-data
redirect_from:
    - /faq/1000-genomes-data-available-genome-browsers/
---

1000 Genomes Project data is available at both [Ensembl](http://www.ensembl.org) and the [UCSC Genome Browser](http://genome.ucsc.edu/).

Ensembl provides consequence information for the variants. The variants that are loaded into the Ensembl database and have consequence types assigned are displayed on the [Variation view](http://www.ensembl.org/Homo_sapiens/Variation/Explore?r=6:73415665-73416665;v=rs311685;vdb=variation;vf=167346058). Ensembl can also offer consequence predictions using their [Variant Effect Predictor (VEP)](http://www.ensembl.org/info/docs/tools/vep/index.html).

You can see individual genotype information in the Ensembl browser by looking at the [Individual Genotypes](http://www.ensembl.org/Homo_sapiens/Variation/Sample?db=core;r=6:73415665-73416665;v=rs311685;vdb=variation;vf=167346058) section of the page from the menu on the left hand side.
