---
title: "Can I get individual genotype information from browser.1000genomes.org?"
faq_tags:
  - data-access
  - ensembl
  - genotypes
  - variants
faq_related:
  - can-i-access-databases-associated-browser
  - are-there-any-scripts-or-apis-use-1000-genomes-data-sets
  - can-i-get-genotypes-specific-individualpopulation-your-vcf-files
  - can-i-get-haplotype-data-1000-genomes-individuals
---
                    
You can see individual genotype information in the browser by looking at the [Individual Genotypes]({{site.browser_url}}/Homo_sapiens/Variation/Individual?r=6:74125388-74126388;v=rs311685;vdb=variation;vf=14071116) section of the a [variant page]({{site.browser_url}}/Homo_sapiens/Variation/Summary?r=6:74125388-74126388;source=dbSNP;v=rs311685;vdb=variation;vf=203479). This can be reached from the menu on the left hand side of the page. You can find a particular variant by putting its rs number in the search box visible at the top right hand corner of every [browser page]({{site.browser_url}}).

The data can be viewed in the same way in [Ensembl](http://www.ensembl.org).
