---
layout: faq
title: "Can I convert vcf files to plink/ped format?"
faq_tags:
  - data-access
  - file-format
  - tools
  - vcf
faq_related:
  - can-i-get-haplotype-data-1000-genomes-individuals
  - are-there-any-fasta-files-containing-1000-genomes-variants-or-haplotypes
  - can-i-get-genotypes-specific-individualpopulation-your-vcf-files
---
                    
<p>We provide a <a href="http://browser.1000genomes.org/Homo_sapiens/UserData/Haploview">vcf to ped tool</a> to convert from vcf to plink format. This tool has documentation for both the&nbsp;<a href="/vcf-ped-converter#Online">web interface</a> and the&nbsp;<a href="/vcf-ped-converter#API%20Script">perl script</a></p>
<p>An example perl command to run the script would be</p>
<pre>perl vcf_to_ped_converter.pl -vcf <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/ALL.chr13.phase1_integrated_calls.20101123.snps_indels_svs.genotypes.vcf.gz" title="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/ALL.chr13.phase1_integrated_calls.20101123.snps_indels_svs.genotypes.vcf.gz">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/ALL.chr13.phase1_integrated_calls.20101123.snps_indels_svs.genotypes.vcf.gz</a> -sample_panel_file <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/phase1_integrated_calls.20101123.ALL.sample_panel" title="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/phase1_integrated_calls.20101123.ALL.sample_panel">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/phase1_integrated_calls.20101123.ALL.sample_panel</a> -region 13:32889611-32973805 -population GBR -population FIN</pre>
