---
title: "Can I map your variant coordinates between different genome assemblies?"
faq_tags:
  - assembly
  - data-analysis
  - tools
  - variants
faq_related:
  - what-tools-can-i-use-to-download-igsr-data
  - can-i-query-igsr-programmatically
redirect_from:
    - /faq/can-i-map-your-snp-coordinates-between-ncbi36-and-grch37/
---

Our pilot data is all presented with respect to NCBI36 and our main project data is all presented with respect to GRCh37. If you need variant calls to be in a particular assembly it is best to go to dbSNP, Ensembl or an equivalent archive using their rs numbers as this will provide a definitive mapping.

If an rs number or equivalent is not available there are tools available to map between NCBI36, GRCh37 and GRCh38 from both [Ensembl](http://www.ensembl.org/Homo_sapiens/UserData/SelectFeatures) and the [NCBI](http://www.ncbi.nlm.nih.gov/genome/tools/remap)
