---
title: "Can I map your snp coordinates between NCBI36 and GRCh37?"
faq_tags:
  - assembly
  - data-analysis
  - tools
  - variants
faq_related:
  - what-tools-can-i-use-download-1000-genomes-data
  - are-there-any-scripts-or-apis-use-1000-genomes-data-sets
---
                    
Our pilot data is all presented with respect to NCBI36 and our main project data is all presented with respect to GRCh37\. If you need variant calls to be in a particular assembly it is best to go to dbSNP or an equivalent archive using their rs numbers as this will provide a definitive mapping.

If an rs number or equivalent is not available there are tools available to map between assemblies from both [ensembl](http://www.ensembl.org/Homo_sapiens/UserData/SelectFeatures) and the [NCBI](http://www.ncbi.nlm.nih.gov/genome/tools/remap)
