---
title: "Can I get individual genotype information from browser.1000genomes.org?"
faq_tags:
  - data-access
  - ensembl
  - genotypes
  - variants
faq_related:
  - can-i-access-databases-associated-browser
  - are-there-any-scripts-or-apis-use-1000-genomes-data-sets
  - can-i-get-genotypes-specific-individualpopulation-your-vcf-files
  - can-i-get-haplotype-data-1000-genomes-individuals
---
                    
The 1000 genomes browser at browser.1000genomes.org but all data is accessible from the Ensembl browser at [grch37.ensembl.org](http://grch37.ensembl.org/index.html) You can see individual genotype information in the browser by looking at the [Sample Genotypes](http://grch37.ensembl.org/Homo_sapiens/Variation/Sample?db=core;r=6:74125388-74126388;v=rs311685;vdb=variation;vf=188547) section of the a [variant page](http://grch37.ensembl.org/Homo_sapiens/Variation/Explore?db=core;r=6:74125388-74126388;v=rs311685;vdb=variation;vf=188547). This can be reached from the menu on the left hand side of the page. You can find a particular variant by putting its rs number in the search box visible at the top right hand corner of every [browser page](http://grch37.ensembl.org/index.html).
