---
title: "Is there gene expression and/or functional annotation available for the samples?"
faq_tags:
  - gene-expression
  - rnaseq
  - sample
  - data-access
  - data-analysis
  - functional-annotation
  - phase-1
faq_related:
  - how-do-i-find-the-most-up-to-date-data
  - can-i-get-phased-genotypes-and-haplotypes-for-the-individual-genomes
  - can-i-get-phased-genotypes-and-haplotypes-for-the-individual-genomes
  - how-do-i-get-a-genomic-region-sub-section-of-your-files
redirect_from:
    - /faq/there-any-functional-annotation-1000-genomes-phase-1-data/
    - /faq/there-any-gene-expression-data-available-1000-genomes-project-samples/
---

**Functional annotation**

As part of our phase 1 analysis we performed functional annotation of our phase 1 variants with respect to both coding and non-coding annotation from [GENCODE](http://www.gencodegenes.org/) and the [ENCODE project](http://www.encodeproject.org/ENCODE/) respectively.

This functional annotation can be found in our [phase 1 analysis results directory](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/). We present both the [annotation](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/functional_annotation/annotation_sets/) we compared the variants to and VCF files which contain the [functional consequences](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/functional_annotation/annotated_vcfs/) for each variant.

**Gene expresssion**

The most important available existing expression datasets involving 1000 Genomes individuals are probably the following:

**RNAseq (mRNA & miRNA) on 465 individuals (CEU, TSI, GBR, FIN, YRI)**

Pre-publication RNA-sequencing data from the Geuvadis project is available through [http://www.geuvadis.org](http://www.geuvadis.org/web/geuvadis/home)

[http://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-1/samples.html](http://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-1/samples.html)  
[http://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-2/samples.html](http://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-2/samples.html)

**RNAseq on 60 CEU individual <sup>[1]</sup>**

[http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-197](http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-197)

**Expression arrays on about 800 HapMap 3 individuals with a lot of overlap with 1000g data <sup>[1,2]</sup>**

[http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-198](http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-198)  
[http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-264](http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-264)

**RNAseq for 69 YRI individuals <sup>[3]</sup>**

[http://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-19480](http://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-19480)

**References**

1.  Reference:Montgomery SB, Sammeth M, Gutierrez-Arcelus M, Lach RP, Ingle C, Nisbett J, Guigo R, Dermitzakis ET. Transcriptome genetics using second generation sequencing in a Caucasian population. Nature. 2010 Apr 1;464(7289):773-7\. Epub 2010 Mar 10.
2.  Reference: Stranger,B.E S.B. Montgomery, A.S. Dimas, L. Parts, O. Stegle, C.E. Ingle, M. Sekowska, G. Davey Smith, D. Evans, M. Gutierrez-Arcelus, A. Price, T. Raj J. Nisbett, A.C. Nica, C. Beazley, R. Durbin, P. Deloukas, E.T. Dermitzakis. Patterns of cis regulatory variation in diverse human populations. PLoS Genetics in press
3.  Reference: Pickrell JK, Marioni JC, Pai AA, Degner JF, Engelhardt BE, Nkadori E, Veyrieras JB, Stephens M, Gilad Y, Pritchard JK. Understanding mechanisms underlying human gene expression variation with RNA sequencing. Nature. 2010 Apr 1;464(7289):768-72\. Epub 2010 Mar 10.

