---
title: "Is there any functional annotation for the 1000 Genomes phase 1 data?"
faq_tags:
  - data-access
  - data-analysis
  - functional-annotation
  - phase-1
faq_related:
  - can-i-get-genotypes-specific-individualpopulation-your-vcf-files
  - how-do-i-get-sub-section-vcf-file
---
                    
As part of our phase1 analysis we performed functional annotation of our phase1 variants with respect to both coding and non coding annotation from [Gencode](http://www.gencodegenes.org/) and the [Encode project](http://www.encodeproject.org/ENCODE/) respectively.

This functional annotation can be found in our [phase1 analysis results directory](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/). We present both the [annotation](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/functional_annotation/annotation_sets/) we compared the variants too and vcf files which contain the [functional consequences](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/functional_annotation/annotated_vcfs/) for each variant.
