---
layout: faq
title: "Is there any functional annotation for the 1000 Genomes phase 1 data?"
faq_tags:
  - data-access
  - data-analysis
  - functional-annotation
  - phase-1
faq_related:
  - can-i-get-genotypes-specific-individualpopulation-your-vcf-files
  - how-do-i-get-sub-section-vcf-file
---
                    
<p>As part of our phase1 analysis we performed functional annotation of our phase1 variants with respect to both coding and non coding annotation from <a href="http://www.gencodegenes.org/">Gencode</a> and the <a href="http://www.encodeproject.org/ENCODE/">Encode project</a> respectively.</p>
<p>This functional annotation can be found in our <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/">phase1 analysis results directory</a>. We present both the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/functional_annotation/annotation_sets/">annotation</a> we compared the variants too and vcf files which contain the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/functional_annotation/annotated_vcfs/">functional consequences</a> for each variant.&nbsp;</p>
