---
layout: faq
title: "What Omni genotype data do you have?"
faq_tags:
  - data-access
  - high-density-genotyping
  - omni
faq_related:
  - what-axiom-genotype-data-do-you-have
  - what-high-density-genotyping-information-do-you-have
  - where-most-recent-release
---
                    
<p>Both Illumina and the Broad institute have carried on genotyping of 1000 genomes and Hapmap samples on the <a href="http://www.illumina.com/dna/omni/index.asp?utm_medium=banners&amp;utm_campaign=omni">Omni Platform</a>. The most recent set of omni genotypes can be found in two places&nbsp;<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/working/20131122_broad_omni/">technical/working/20131122_broad_omni</a>&nbsp;and&nbsp;<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/working/20130812_sanger_omni/">technical/working/20130812_sanger_omni/</a>&nbsp;These contain GRCh37 based vcf files for the chip and normalised and raw intensity files.</p>
<p>We also used omni data as part of our phase1 analysis. The set used for this can be found in&nbsp;<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/omni_haplotypes/" style="text-align: justify;">phase1/analysis_results/supporting/omni_haplotypes/</a>. These are vcf files mapped to GRCh37. There is a single file which contains all the genotypes on the chip and then fully phased vcf files per chromosome. The haplotype estimation was carried out using the <a href="http://www.shapeit.fr/" style="text-align: justify;">ShapeIt2</a> algorithm.</p>
