---
layout: faq
title: "What is a sequence index file?"
faq_tags:
  - data-access
  - meta-data
  - sequence
faq_related:
  - what-do-names-your-fastq-files-mean
  - how-much-sequence-data-has-been-generated-single-individuals
  - where-are-your-sequence-files-located
---
                    
<p>We describe our sequence meta data in sequence.index files. The most recent version of this file is always found at the head of the ftp site called <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/sequence.index">sequence.index</a>. We also place dated versions of these files in the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/sequence_indices/">sequence_indices</a> directory. The date always follows the form YYYYMMDD. The date represents a particular freeze of sequence data and you will also see this same date using in alignment and vcf files to represent analysis products built from this sequence freeze.</p>
<p>These files are tab delimited files containing <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/README.sequence_data">these columns</a></p>
<p>1. FASTQ_FILE, path to fastq file on ftp site<br />
<!-- br -->
2. MD5, md5sum of file<br />
<!-- br -->
3. RUN_ID, SRA/ERA run accession <br />
4. STUDY_ID, SRA/ERA study accession <br />
5. STUDY_NAME, Name of stury <br />
6. CENTER_NAME, Submission centre name <br />
7. SUBMISSION_ID, SRA/ERA submission accession <br />
8. SUBMISSION_DATE, Date sequence submitted, YYYY-MM-DAY <br />
9. SAMPLE_ID, SRA/ERA sample accession <br />
10. SAMPLE_NAME, Sample name <br />
11. POPULATION, Sample population, this is a 3 letter code and it is defined in <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/README.populations">README.populations</a><br />
12. EXPERIMENT_ID, Experiment accession <br />
13. INSTRUMENT_PLATFORM, Type of sequencing machine <br />
14. INSTRUMENT_MODEL, Model of sequencing machine <br />
15. LIBRARY_NAME, Library name <br />
16. RUN_NAME, Name of machine run <br />
17. RUN_BLOCK_NAME, Name of machine run sector (This is no longer recorded so this column is entirely null, it was left in so as not to disrupt existing sequence index parsers) <br />
18. INSERT_SIZE, Submitter specifed insert size <br />
19. LIBRARY_LAYOUT, Library layout, this can be either PAIRED or SINGLE <br />
20. PAIRED_FASTQ, Name of mate pair file if exists (Runs with failed mates will have a library layout of PAIRED but no paired fastq file) <br />
21. WITHDRAWN, 0/1 to indicate if the file has been withdrawn, only present if a file has been withdrawn <br />
22. WITHDRAWN_DATE This is generally the date the file is generated on <br />
23. COMMENT, comment about reason for withdrawal <br />
24. READ_COUNT, read count for the file <br />
25. BASE_COUNT, basepair count for the file <br />
26. ANALYSIS_GROUP, the analysis group of the sequence, this reflects sequencing strategy. Currently this includes low coverage, high coverage, exon targetted and exome to reflect the 2 non low coverage pilot sequencing stratergies and the 2 main project sequencing stratergies used by the 1000 genomes project.
<!-- br -->
</p>
