---
title: "What sequencing platforms and methods were used by different projects within IGSR?"
faq_tags:
  - data-access
  - statistics
faq_related:
  - are-there-any-statistics-about-how-much-sequence-data-is-in-igsr
  - how-much-disk-space-used-1000-genomes-project
  - how-many-individuals-have-been-sequenced-in-igsr-projects-and-how-were-they-selected
  - about-index-files
  - about-alignment-files-bam-and-cram
redirect_from:
    - /faq/what-sequencing-platforms-were-used-1000-genomes-project/
    - /faq/what-library-insert-sizes-where-used-1000-genomes-project/
    - /faq/what-read-lengths-are-being-used-project/
---

Data in IGSR spans a wide range of technologies.

The technologies used in recent work are listed in the [data portal](/data-portal/sample) and are visible in 'Technology view', although this does not include older technologies used in, for example, the pilot phase of the 1000 Genomes Project. The portal also enables filtering of data sets by technology.

The most common form of data is Illumina genomic seqeunce data. However, the number of samples with long read data from PacBio and Oxford Nanopore is growing. 

Further detail on the technologies present in our collection can be found in the [accompanying publications](/data-portal/data-collection) for the given collection.
