---
title: "What capture technology did the Exome sequencing use?"
faq_tags:
  - capture
  - data-access
  - exome
  - pull-down
faq_related:
  - what-are-targets-your-whole-exome-sequencing
  - what-are-targets-your-exon-targetted-pilot-study
  - what-difference-between-analysis-groups-exome-and-exon-targetted-sequence-index
---
                    
Different centres have used different pull-down technologies for the Exome sequencing done for the 1000 Genomes project.

Baylor College of Medicine used [NimbleGen SeqCap_EZ_Exome_v2](http://www.nimblegen.com/products/seqcap/ez/v2/index.html) for its Solid based exome sequencing. For its more recent Illumina based exome sequencing it used a custom array [HSGC VCRome](http://www.nimblegen.com/products/seqcap/ez/designs/)

The Broad Institute has used Agilent SureSelect_All_Exon_V2 ([https://earray.chem.agilent.com/earray/](https://earray.chem.agilent.com/earray/){:target="_blank"} using ELID: S0293689).

The BGI used <span>NimbleGen SeqCap EZ exome V1</span> for the phase 1 samples and [NimbleGen SeqCap_EZ_Exome_v2](http://www.nimblegen.com/products/seqcap/ez/v2/index.html) for phase 2 and 3 (the v1 files were obtained from BGI directly; they are discontinued from Nimblegen).

The Washington University Genome Center used Agilent SureSelect_All_Exon_V2 ([https://earray.chem.agilent.com/earray/](https://earray.chem.agilent.com/earray/){:target="_blank"} using ELID: S0293689) for phase 1 and phase 2, and [NimbleGen SeqCap_EZ_Exome v3](http://www.nimblegen.com/products/seqcap/ez/v3/index.html) for phase 3
