---
layout: faq
title: "What capture technology does the Exome sequencing used?"
faq_tags:
  - capture
  - data-access
  - exome
  - pull-down
faq_related:
  - what-are-targets-your-whole-exome-sequencing
  - what-are-targets-your-exon-targetted-pilot-study
  - what-difference-between-analysis-groups-exome-and-exon-targetted-sequence-index
---
                    
<p>Different centres have used different pull down technologies for the Exome sequencing done for the 1000 Genomes project.</p>
<p>Baylor College of Medicine used <a href="http://www.nimblegen.com/products/seqcap/ez/v2/index.html">NimbleGen SeqCap_EZ_Exome_v2</a> for its Solid based exome sequencing. and for its more recent Illumina based exome sequencing it used a custom array <a href="http://www.nimblegen.com/products/seqcap/ez/designs/">HSGC VCRome</a></p>
<p>The Broad Institute has used&nbsp;&nbsp;Agilent SureSelect_All_Exon_V2 (<a href="https://earray.chem.agilent.com/earray/" target="_blank">https://earray.chem.agilent.com/earray/</a>&nbsp;using ELID: S0293689).</p>
<p>The BGI:&nbsp;<span>NimbleGen SeqCap EZ exome V1</span>&nbsp;for the phaseI samples and <a href="http://www.nimblegen.com/products/seqcap/ez/v2/index.html">NimbleGen SeqCap_EZ_Exome_v2</a>&nbsp; for phase II and III&nbsp;(The v1 files obtained from BGI directly; they are discontinued from Nimblegen).</p>
<p>The Washington University Genome Center used Agilent SureSelect_All_Exon_V2 (<a href="https://earray.chem.agilent.com/earray/" target="_blank">https://earray.chem.agilent.com/earray/</a> using ELID: S0293689) for phaseI and phaseII and&nbsp;<a href="http://www.nimblegen.com/products/seqcap/ez/v3/index.html">NimbleGen SeqCap_EZ_Exome v3</a>&nbsp;for phaseIII</p>
