---
layout: faq
title: "What is a sequence index file?"
faq_tags:
  - data-access
  - meta-data
  - sequence
faq_related:
  - what-do-names-your-fastq-files-mean
  - how-much-sequence-data-has-been-generated-single-individuals
  - where-are-your-sequence-files-located
---
                    
<p>We describe our sequence meta data in sequence index files. The index for data from the 1000 Genomes project can be found in the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000_genomes_project/">1000 Genomes data collection directory</a>. Additional indices are present for data in other data collections. Our old index files which describe the data used in the main project can be found in the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/historical_data/former_toplevel/">historical_data directory</a></p>
<p>Sequence index files are tab delimited files and frequently contain these columns 1. FASTQ_FILE, path to fastq file on ftp site or ENA ftp site<br />
2. MD5, md5sum of file<br />
3. RUN_ID, SRA/ERA run accession<br />
4. STUDY_ID, SRA/ERA study accession <br />
5. STUDY_NAME, Name of study<br />
6. CENTER_NAME, Submission centre name <br />
7. SUBMISSION_ID, SRA/ERA submission accession <br />
8. SUBMISSION_DATE, Date sequence submitted, YYYY-MM-DAY <br />
9. SAMPLE_ID, SRA/ERA sample accession <br />
10. SAMPLE_NAME, Sample name <br />
11. POPULATION, Sample population, this is a 3 letter code and it is defined in README_populations.md<br />
12. EXPERIMENT_ID, Experiment accession <br />
13. INSTRUMENT_PLATFORM, Type of sequencing machine <br />
14. INSTRUMENT_MODEL, Model of sequencing machine <br />
15. LIBRARY_NAME, Library name <br />
16. RUN_NAME, Name of machine run <br />
17. RUN_BLOCK_NAME, Name of machine run sector (This is no longer recorded so this column is entirely null, it was left in so as not to disrupt existing sequence index parsers)<br />
18. INSERT_SIZE, Submitter specified insert size<br />
19. LIBRARY_LAYOUT, Library layout, this can be either PAIRED or SINGLE <br />
20. PAIRED_FASTQ, Name of mate pair file if exists (Runs with failed mates will have a library layout of PAIRED but no paired fastq file) <br />
21. WITHDRAWN, 0/1 to indicate if the file has been withdrawn, only present if a file has been withdrawn <br />
22. WITHDRAWN_DATE This is generally the date the file is generated on <br />
23. COMMENT, comment about reason for withdrawal <br />
24. READ_COUNT, read count for the file <br />
25. BASE_COUNT, basepair count for the file <br />
26. ANALYSIS_GROUP, the analysis group of the sequence, this reflects sequencing strategy. For 1000 Genomes project data, this includes low coverage, high coverage, exon targeted and exome to reflect the 2 non low coverage pilot sequencing strategies and the 2 main project sequencing strategies used by the 1000 genomes project.</p>
