---
layout: faq
title: "What capture technology does the Exome sequencing used?"
faq_tags:
  - capture
  - data-access
  - exome
  - pull-down
faq_related:
  - what-are-targets-your-whole-exome-sequencing
  - what-are-targets-your-exon-targetted-pilot-study
  - what-difference-between-analysis-groups-exome-and-exon-targetted-sequence-index
---
                    
Different centres have used different pull down technologies for the Exome sequencing done for the 1000 Genomes project.

Baylor College of Medicine used [NimbleGen SeqCap_EZ_Exome_v2](http://www.nimblegen.com/products/seqcap/ez/v2/index.html) for its Solid based exome sequencing. and for its more recent Illumina based exome sequencing it used a custom array [HSGC VCRome](http://www.nimblegen.com/products/seqcap/ez/designs/)

The Broad Institute has used  Agilent SureSelect_All_Exon_V2 ([https://earray.chem.agilent.com/earray/](https://earray.chem.agilent.com/earray/){:target="_blank"} using ELID: S0293689).

The BGI: <span>NimbleGen SeqCap EZ exome V1</span> for the phaseI samples and [NimbleGen SeqCap_EZ_Exome_v2](http://www.nimblegen.com/products/seqcap/ez/v2/index.html)  for phase II and III (The v1 files obtained from BGI directly; they are discontinued from Nimblegen).

The Washington University Genome Center used Agilent SureSelect_All_Exon_V2 ([https://earray.chem.agilent.com/earray/](https://earray.chem.agilent.com/earray/){:target="_blank"} using ELID: S0293689) for phaseI and phaseII and [NimbleGen SeqCap_EZ_Exome v3](http://www.nimblegen.com/products/seqcap/ez/v3/index.html) for phaseIII
