---
layout: faq
title: "Where can I get consequence annotations for the 1000 genome variants?"
faq_tags:
  - annotation
  - consequence
  - data-access
  - ensembl
  - variants
faq_related:
  - where-are-your-variant-files-located
  - how-do-i-get-sub-section-vcf-file
  - are-there-any-scripts-or-apis-use-1000-genomes-data-sets
---
                    
<p>The 1000 genomes phase1 analysis calculated consequences based on <a href="http://www.gencodegenes.org/releases/7.html">Gencode annotation</a>&nbsp;and this can be found in the directory&nbsp;<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/functional_annotation/">phase1/analysis_results/functional_annotation/</a></p>
<p>Ensembl can also provides consequence information for the variants. The variants that are loaded into the Ensembl database and have consequence types assigned and displayed on the <a href="http://browser.1000genomes.org/Homo_sapiens/Variation/Mappings?db=core;r=6:74125388-74126388;v=rs311685;vdb=variation;vf=14071116" target="_blank">Variation view</a>. Ensembl can also offer consequence predictions using their <a href="http://browser.1000genomes.org/Homo_sapiens/UserData/UploadVariations?db=core" target="_blank">Variant effect predictor</a>.</p>
<p>Please note the phase1 annotations and the Ensembl annotations visible via the browser due to using different versions of gene and non coding annotation.</p>
