---
layout: faq
title: "Where are your alignment files located?"
faq_tags:
  - alignment
  - bam
  - data-access
faq_related:
  - how-are-your-alignments-generated
  - what-format-are-your-alignments-and-what-do-names-mean
  - how-do-i-get-sub-section-vcf-file
  - which-reference-assembly-do-you-use
---
                    
<p>Our main alignment files are located in our <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data/">data directory</a>. Our mapped bams contain reads which aligned to the whole genome.</p>
<p>You can find an index of our alignments in our <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/alignment.index">alignment.index file</a>. There are dated versions of these files and statistics surrounding each alignment release in the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/alignment_indices/">alignment_indicies</a> directory. Please note with few exceptions we only keep the most recent QC passed alignment for each sample on the ftp site.</p>
<p>We also have frozen versions of the alignments use for both the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/">pilot</a> and the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/">phase1</a> analyses in different directories on the ftp site. Please note the that the pilot alignments are mapped to NCBI36 rather than GRCh37 like all other alignments on the ftp site.</p>
