---
layout: faq
title: "Which reference assembly do you use?"
faq_tags:
  - assembly
  - data-access
  - data-analysis
  - grch37
  - ncbi36
  - reference
faq_related:
  - how-are-your-alignments-generated
  - what-format-are-your-alignments-and-what-do-names-mean
  - where-are-your-alignment-files-located
---
                    
<p>The reference assembly the 1000 genomes project has mapped sequence data to has changed over the course of the project.</p>
<p>For the pilot phase we mapped data to <span>NCBI36</span>. A copy of our reference <span>fasta</span> file can be found on the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/technical/reference/" target="_blank">ftp site</a>.</p>
<p>For the <span>phase1</span> analysis we mapped to <span>GRCh37</span>. Our <span>fasta</span> file which can be found <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/" target="_blank">here</a> called <span>human_g1k_v37.fasta.gz</span>, it contains the <span>autosomes</span>, X, Y and MT but no <span>haplotype</span> sequence or <span>EBV</span></p>
<p>For the final round of alignments the sequence data will be mapped to a set of sequences derived from the <span>GRCh37</span> assembly. &nbsp;This <span>GRCh37-derived</span> alignment set includes chromosomal plus <span>unlocalized</span> and unplaced <span>contigs</span>, the <span>rCRS</span> mitochondrial sequence (AC:NC_012920), Human <span>herpesvirus</span> 4 type 1 (AC:NC_007605) and decoy sequence derived from <span>HuRef</span>, Human <span>Bac</span> and <span>Fosmid</span> clones and <span>NA12878</span>. These files are available in <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/" target="_blank"><span>phase2_reference_assembly_sequence</span></a> on the ftp site. &nbsp;All human variant coordinates reported by the 1000 Genomes project are in <span>GRCh37</span> coordinates.</p>
