---
layout: faq
title: "Why is only 85% of the genome assayable?"
faq_tags:
  - data-analysis
  - sequence
faq_related:
  - how-are-your-alignments-generated
  - what-does-individual-have-genotype-location-where-it-has-no-sequence-coverage
  - what-depth-coverage-your-phase1-variants
---
                    
<p><span class="il">For both the pilot project and the phase1 analysis the 1000genomes project created what they defined as accessibilty masks.&nbsp;</span></p>
<p><span class="il">The pilot mask showed that only 85</span>% of the genome is accessible to accurate analysis with the short read technology used by the 1000 Genomes pilot project. &nbsp;The remaining 15% is either repeats or segmental duplications. There is more information about the pilot mask in <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/release/2010_03/pilot1/supporting/README_callability_masks">README.callability_masks</a><br />
<br />
For the phase1 analysis using the pilot callability criteria 94% of the genome was accessible. A stricter mask was also created for the phase1 project to be used for population genetics analysis, This mask used a narrower band of coverage criteria and also insisted that less than 0.1% of reads have a mapping quality of 0 and the average mapping quality should be 56 or higher. These criteria lead to 72.2% of the genome being accessible to accurate analysis with the short read technology used by the 1000 Genomes project.</p>
