---
layout: angularjs_partial
---

The Human Genome Structural Variation Consortium (HGSVC), funded by NHGRI, have built on their [earlier work](https://www.internationalgenome.org/data-portal/data-collection/structural-variation) exploring multiple technologies for structural variation discovery.

In HGSVC2, their work applies multiple technologies to samples across the 1000 Genomes Project populations, producing haplotype resolved assemblies and identifying structural variation. Detected variation has been genotyped across the 3,202 1000 Genomes Project samples for which high-coverage Illumina data is available. The Illumina data used in this work is the [30x data generated by NYGC](https://www.internationalgenome.org/data-portal/data-collection/30x-grch38).

Data generated by the project is presented below. All working files and current genotype files (VCFs) are available on our [FTP site](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/). 

Some key data sets include:

* **Merged variation calls** - [Multisample merged variation call files](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v1.0/integrated_callset/) contain calls from variation discovery in the samples for which assemblies were generated. A [description](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v1.0/integrated_callset/README_20200814_Freeze3) of the multiple formats in which the data is shared is also provided. The VCF versions of these files are listed below.
* **Genotypes across the 3202 samples** - [PanGenie calls](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v1.0/PanGenie_results/) genotype the variation detected in the assemblies across the 3202 samples in the 1000 Genomes Project sample collection for which high-coverage Illumina data is available. The [quality filters](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v1.0/PanGenie_filters/) used are also shared.
* **eQTL and sQTL analyses** - results from [expression-QTL](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v1.0/eQTL_V4_Dec_2020/) and [splice-QTL](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v1.0/sQTL_V4_Dec_2020/) analyses are provided.
* **Assemblies** - The generated [assemblies](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v1.0/assemblies/), which were then used in subsequent analysis, are also available.
* **Reference data** - The [version of GRCh38 used in this work](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/technical/reference/20200513_hg38_NoALT/), [files detailing low confidence regions of the genome excluded from analysis](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/technical/filter/20210127_LowConfidenceFilter/) and [technical supporting](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/technical/tech-support-files/) files (such as configuration files) are included.
